sections/03-methods.md

```diff
@@ -0,0 +1,65 @@
+# Methods
+
+## Disciplines, Interdisciplinarity, and Adjacency
+
+It is difficult to address the question of methods without also touching on the issue of disciplines and disciplinarity, and in the context of this particular work, of interdisciplinarity. My artwork and research practice here is engaged in, amongst, and alongside three ostensibly distinct disciplines: video art, music, and design<sup>1</sup>. Within these disciplines, my activities can be further segmented into the categories of digital cinema, computer music, and inclusive software architecture. Yet such categories sit slightly uncomfortably, and the question that invariably arises from a confession of multiple disciplinary affinities is: What, then, is the relationship between these disciplines in my artistic and scholar practice?
+
+First, I want to be clear that I am concerned here with affording a certain degree of autonomy--a slight gap--between my design, music, and video practices. This, is not, of course, to argue that there are no significant connections between these activities, but rather to retain, as best as possible, the unique medial and conceptual qualities that inhere in each, while opening up the possibility for something new to emerge from their proximity. Music, for example, has a somewhat "spectral" presence within this particular project, since the end product consists only of a collection of videos, many of which are completely silent. Throughout the process of making (mostly) silent videos, however, I was consistently composing electronic music, improvising on the guitar, and designing and supporting open source music synthesis software. So if my musical activities are to be felt within these videos--or, for that matter, my technical/design activities within either side of my artistic practice--then I would like it not to be understood within a mode of "integrated practice," nor one that aims for a synthesis of media, disciplines, and methods, but rather one in which a small network or assemblage of parallel irritations and transformations move autonomously in and out of phase with one another. Indeed, it is this idea of systemic closure and transformation, rather than synthesis--which I call _disciplinary adjacency_--that serves as the "metamethod" upon which my iterative practice/research methods of lensing and open source community collaboration are based. The point of disciplinary adjacency is not to argue for a separation of practices or a lack of relations, but to suggest that something interesting is produced as a result of maintaining the distinction. Disciplinary relations are emergent within the space of a particular practice, and they thus have the ability to re-form and be actively reconfigured over time and in different contexts<sup>2</sup>.
```

Comment:
Stray comma at "This, is not"

Can things be considered adjacent if one hasn't fully accepted the distinctions they are based on?
Is interdisciplinarity, as traditionally conceived, always aimed at the end of synthesis?
What is the sense of "closure" in "systemic closure"?
Making a collocation out of "irritations and transformations" seems a bit German ... since this sentence is the centrepiece of the paragraph I think it could be illuminated a little more.